CHANGELOG.md

```diff
@@ -8,84 +8,13 @@ Commits have to follow following convention: https://www.conventionalcommits.org
 
 ### Bug Fixes
 
-- Pcan logs for macos
-
-### Miscellaneous Tasks
-
-- Bump pykiso-python-uds from 3.0.0 to 3.0.1 ([#163](https://github.com/orhun/git-cliff/issues/163))
-- Bump pytest from 7.1.2 to 7.1.3 ([#155](https://github.com/orhun/git-cliff/issues/155))
-- Debian:11.0
-- Bump black from 22.3.0 to 22.8.0 ([#154](https://github.com/orhun/git-cliff/issues/154))
-
-## Version 0.19.3 (2022-09-16)
-
-### Bug Fixes
-
-- Handle case odx_file_path parameter is type of bool ([#158](https://github.com/orhun/git-cliff/issues/158))
-- Step report problem with inspect module in python>=3.8 ([#160](https://github.com/orhun/git-cliff/issues/160))
-
-### Miscellaneous Tasks
-
-- Release 0.19.3 ([#161](https://github.com/orhun/git-cliff/issues/161))
-
-### New Features
-
-- Write stderr to file when file logging is activated ([#159](https://github.com/orhun/git-cliff/issues/159))
-
-## Version 0.19.2 (2022-09-13)
-
-### Bug Fixes
-
-- Changelog only triggered on master
-- Cleanup auxiliary properly by removing them from sys.modules ([#147](https://github.com/orhun/git-cliff/issues/147))
-- Fix double Logging with --junit option
-- Update logging initializer so that internal logs are always logged in file
-
-### Miscellaneous Tasks
-
-- Add jinja2 for step reporter dependency
-- [**breaking**] Release 0.19.2
-
-### New Features
-
-- Select test case with regex ([#146](https://github.com/orhun/git-cliff/issues/146))
-
-### Refactorings
-
-- Small fixes
-
-## Version 0.19.1 (2022-09-02)
-
-### Bug Fixes
-
-- Readthedocs requirements
-
-### Miscellaneous Tasks
-
-- Bump pre-commit from 2.19.0 to 2.20.0
-- Bump coverage from 5.5 to 6.4.4
-- Bump pytest-mock from 3.7.0 to 3.8.2
-- Bump pylink-square from 0.12.0 to 0.14.2 ([#137](https://github.com/orhun/git-cliff/issues/137))
-- Update click ([#143](https://github.com/orhun/git-cliff/issues/143))
-- Release 0.19.1 ([#144](https://github.com/orhun/git-cliff/issues/144))
-
-### New Features
-
-- Add start and stop uds tester present sender ([#131](https://github.com/orhun/git-cliff/issues/131))
-- Add cc_serial connector ([#124](https://github.com/orhun/git-cliff/issues/124))
-
-### Refactorings
-
-- Restructure the documentation
-
-### Ci
-
-- Replace auto-changelog
-
-## Version 0.19.0 (2022-08-30)
-
-### Bug Fixes
-
+- Wait for logger thread to quit gracefully before closing segger channel ([#28](https://github.com/orhun/git-cliff/issues/28))
```

Comment:
I have really no idea why this started... https://github.com/BKaDamien/kiso-testing/blob/Ensure_default_behaviour_when_(suite_case)_id_is_0/.github/workflows/changelog.yml#L4 is not fulfilled at all!

Reply:
I dont even touch it. I now every pull make a rebase instead of merge

Reply:
Yes, ignore it. Should be automatically fixed on master, after merge